packages/pvl/lib/patterns.js

```diff
@@ -4,13 +4,13 @@ module.exports = {
   // Currently doesn't handle other PVL-allowed numeric notations
   // For example, based notation (`16#4B#`) or scientific notation (`1.234E2`)
   numericPatterns: [
-    /^([+\-]?\d+(\.\d*)?)$/,
+    /^([+\-]?(([1-9]\d*)|0)(\.\d*)?)$/,
```

Comment:
So the change here is that if it's not zero, it has to start with a non-zero value -- what's the reason for this?

Reply:
Javascript parses `006` into a number with value `6`, so we want leading-zero cases to be a string instead.